### PR TITLE
docs(ecopass-kit): Updated the EcoPass KIT Adoption View

### PR DESCRIPTION
**Pull request by matbmoser**
## Why do we create this PR?

We want to update the adoption view from the Eco Pass kit for release 24.05.

## What we want to achieve with this PR?

Have an adoption view with the latest models and descriptions. With the images and diagrams corrected and ready for the IP check.

### Added
- Added reference to standards in the text to reference the development view standard list.
- Added e-drive payload to the ecopass kit adoption view
- Added option to hide the payloads in the adoption view

### Updated
- Updated the business architecture diagram, removing the versions and extra passports
- Updated the value chain diagram, removing the empty space in the left corner of the image.
- Updated adoption view descriptions with new chapters

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs. (NO DEPENDENCIES)
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files
- [x] LINTING issues are resolved.
- [x] CHANGELOG updated

### CODE REVIEW

**Review by LuisRickert — APPROVED** [review]

**Review by LuisRickert — APPROVED** [review]

**Comment by matbmoser** [comment]
In teory the linting shall be fixed now

**Review by danielmiehle — APPROVED** [review]
LGTM

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md [hunk 1]
```diff
@@ -112,365 +112,590 @@ By providing a standardized, scalable and easy-to-adopt solution, the EcoPass KI
...
+The data model in version 4.0.0 corresponds to the Digital Product Passport information required by the proposed Ecodesign Regulation ([ESPR-2022](<(https://commission.europa.eu/energy-climate-change-environment/standards-tools-and-labels/products-labelling-rules-and-requirements/sustainable-products/ecodesign-sustainable-products_en)>)) from March 30th, 2022. In the future, delegated acts for specific product groups would require a product passport to be available for each product. The data model will be updated, as newer versions of the regulation will be published. The latest version of the document was the [provisional agreement](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CONSIL:ST_5147_2024_INIT) between the EU Council and the Parliament from January 9th, 2024. The text is informal, but the content of the final regulation was agreed between these two institutions.
```
Comment (HFocken) [inline docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:127]: Please verify that this is still the latest version of published regulation - there is a new delegated act published by the EC on EV batteries.

**Review by HFocken — COMMENTED** [review]
Thank you for the great addition - only one comment on current regulation - please verify!

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md (hunk 1 above)
Comment (catroest) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md]: This is the regulation about  Digital Product Passports (ESPR = Ecodesign for Sustainable Products Regulation). This data model correcponds to the indicated version of the document.

Comment (HFocken) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md]: Thanks for clarifying!

**Review by HFocken — APPROVED** [review]
LGTM

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md [hunk 2]
```diff
@@ -112,365 +112,590 @@ By providing a standardized, scalable and easy-to-adopt solution, the EcoPass KI
...
 
-### Battery Pass
+Other specific product passports which are further described in the EcoPass KIT are the [battery](#battery-passport), [transmission](#transmission-passport) and [electric drive](#electric-drive-passport). All of them depend on the generic Digital Product Passport data model. Further data models will follow.
 
-#### Battery Pass Introduction
+The goal of these passports is to create unified data models that can be used and adapted across the automotive industry. These data models are modelled according to the [CX-0003](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Archiv/Update_Juli_23_R_3.2/CX-0003-SAMMSemanticAspectMetaModel-v.1.0.2.pdf) BAMM/SAMM Standard in Version 1.1.0 (updated link once published). For detailed information please follow the embedded links to the Catena-X Standard library.
```
Comment (jSchuetz88) [inline docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md:119]: As the link to the PDF-file of CX-0003 will be deprecated with the Release 24.05, I'd suggest to change the link to: https://catena-x.net/en/standard-library

PS: As BAMM models are deprecated with CX-0003:1.1 and only SAMM models are accepted in accordance to CX-0003, the part with "_BAMM/SAMM Standard_" should be changed in "_SAMM Standard_"

**Review by jSchuetz88 — APPROVED** [review]
Thank you for the update of the KIT, only a minor remark from my side.

File: docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md (hunk 2 above)
Comment (matbmoser) [reply docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md]: Done

**Review by danielmiehle — DISMISSED** [review]
LGTM

**Review by maximilianong — DISMISSED** [review]

**Comment by danielmiehle** [comment]
@matbmoser There are some new conflicting files. Would you be so kind to fix it?

**Comment by matbmoser** [comment]
@danielmiehle done!

**Comment by matbmoser** [comment]
@danielmiehle lets merge the #789 PR first then I solve the changelog linting issues while resolving the merge

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Pull request merged**